integration-tests/ledger-signer.spec.ts

```diff
@@ -164,10 +164,10 @@ CONFIGS().forEach(({ lib, setup, rpc }) => {
       });
     })
 
-    describe('Should be able to used Ledger with wallet API', () => {
+    describe('Test an ability to use Ledger with wallet api', () => {
       jest.setTimeout(120000)
 
-      it('Should sign and inject transaction with Ledger', async (done) => {
+      it('As a User I should be able to sign and inject transaction with Ledger', async (done) => {
```

Comment:
Should we use the same format as the other files "Verify that..." without "As a User I want to"

Reply:
@michaelkernaghan